robot-server/tests/integration/fixtures.py

```diff
@@ -27,3 +28,27 @@ def check_health_response(response: Response) -> None:
     }
 
     assert response.json() == expected
+
+
+def delete_all_runs(response: Response, host: str, port: str) -> None:
+    """Intake the response of a GET /runs and delete all runs if any exist."""
+    headers = {"Opentrons-Version": "*"}
+    base_url = f"{host}:{port}"
+    try:
+        runs = response.json()["data"]
+        if runs == []:
+            return
+        run_ids = [run["id"] for run in runs]
+        for run_id in run_ids:
+            delete_response = requests.delete(
+                f"{base_url}/runs/{run_id}", headers=headers
+            )
+            print(
+                f"""Delete run {run_id}
+                response status code = {delete_response.status_code}
+                """
+            )
```

Comment:
I think this will print weirdly because the indentation is inside the string. I'd do:

```suggestion
            print(
                f"Deleted run {run_id},"
                f" response status code = {delete_response.status_code}"
            )
```

Reply:
Looks a little tidier.